test/utils.py

```diff
@@ -188,6 +188,25 @@ def gnoi_reboot(method, delay, message):
     ret, msg = run_cmd(cmd)
     return ret, msg
 
+def gnoi_kill_process():
+    path = os.getcwd()
+    json_data = '{"name": "snmp"}' 
```

Comment:
Why do you use fixed data for kill process? I think gnoi_kill_process and gnoi_restart_process should be the same.

Reply:
Thanks, changed to match gnoi_restart_process